backend/census_historical_migration/test_federal_awards_xforms.py

```diff
@@ -588,6 +589,41 @@ def test_match_numbers_mixed_empty_and_non_empty(self):
         self.assertEqual(transformed_ids, expected_ids)
 
 
+class TestXformMissingMajorProgram(SimpleTestCase):
+    class AuditMock:
+        def __init__(
+            self,
+            major_program,
+            audit_type,
+        ):
+            self.MAJORPROGRAM = major_program
+            self.TYPEREPORT_MP = audit_type
+
+    def test_xform_normal_major_program(self):
+        """Test for normal major program"""
+        audits = [self.AuditMock("Y", "U")]
+
+        xform_missing_major_program(audits)
+
+        self.assertEqual(audits[0].MAJORPROGRAM, "Y")

```

Comment:
Shouldn't the result of the transformation be checked here?

Reply:
Some xforms modify the audit data directly, as `xform_missing_major_program` does

Reply:
How do we decide which xforms return the modified data and which modify in place? Wondering if we might run into issues if/when we do stuff [like this](https://github.com/GSA-TTS/FAC/blob/967ddfa6b304304930ed0015842aa5f8ec76ca40/backend/audit/intakelib/transforms/runners.py#L56), which appears to assume that that there's data being returned from each function

Reply:
A fair question, though the xforms we've been adding haven't been a part of those runner lists. A lot of the ones I was seeing altered the data instead of returning a value, so that's what I rolled with 🤷. Not opposed to standardizing our approach, though.